src/main/java/core/basesyntax/service/strategy/ActivityStrategy.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.strategy;
+
+import core.basesyntax.service.strategy.impl.BalanceService;
+import core.basesyntax.service.strategy.impl.PurchaseService;
+import core.basesyntax.service.strategy.impl.ReturnService;
+import core.basesyntax.service.strategy.impl.SupplyService;
+import core.basesyntax.utility.FruitTransaction;
+
+public class ActivityStrategy {
```

Comment:
Same here, this is an `OperationStrategy` because it provides Handlers based on the `Operation` of the transaction, do not forget to correct the method here respectively